commons/che-core-commons-gwt/src/main/java/org/eclipse/che/ide/util/MathUtils.java

```diff
@@ -1,24 +1,50 @@
-// Copyright 2012 Google Inc. All Rights Reserved.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
+/*******************************************************************************
+ * Copyright (c) 2012-2015 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
 package org.eclipse.che.ide.util;
 
-/** Utility methods for math-related functionality. */
-public class MathUtils {
+import static java.lang.Math.pow;
+
+/**
+ * Provides utility methods that helps math computations.
+ *
+ * @author Vlad Zhukovskiy
+ */
+public final class MathUtils {
+    /**
+     * A Bézier curve is a parametric curve frequently used in computer graphics and related fields.
+     * In vector graphics, Bézier curves are used to model smooth curves that can be scaled indefinitely.
+     * <p/>
+     * Four points x1, x2, x3 and x4 in the plane or in higher-dimensional space define a cubic Bézier curve.
+     * The curve starts at x1 going toward x2 and arrives at x4 coming from the direction of x3.
+     * Usually, it will not pass through x2 or x3; these points are only there to provide directional information.
+     * The distance between x1 and x2 determines "how far" and "how fast" the curve moves towards x2 before turning towards x3.
+     * <p/>
+     * https://en.wikipedia.org/wiki/B%C3%A9zier_curve
+     *
+     * @param t
+     *         should be between 0.0 and 1.0
+     * @param x1
+     *         coordinate for the x1
+     * @param x2
+     *         coordinate for the x2
+     * @param x3
+     *         coordinate for the x3
+     * @param x4
+     *         coordinate for the x4
+     * @return coordinate for the point
+     */
+    public static double getCubicBezier(double t, double x1, double x2, double x3, double x4) {
```

Comment:
If somebody will create another functionality that uses for example this function, he will know that in commons module there is a Math utility class instead of searching for it inside other modules.